script.service.ltvpl/addon.xml

```diff
@@ -0,0 +1,57 @@
+<addon id="script.service.ltvpl" name="Live TV Playlist" provider-name="jmooremcc" version="1.0.0">
+  <requires>
+    <import addon="xbmc.python" version="2.26.0" />
+    <import addon="xbmc.addon" version="17.0.0" />
```

Comment:
`xbmc.addon` dependency not needed, please remove

Reply:
Removed